source/docs/zero-to-robot/step-3/openmesh.rst

```diff
@@ -0,0 +1,231 @@
+.. include:: <isonum.txt>
+
+
+# Using the OpenMesh OMP5 Radio
+
+
+Prior to 2025 the official radio used in FRC was an OM5P-AN or OM5P-AC Radio.
+As of the 2025 season  The OpenMesh radio is still being used in Taiwan and China.
+
+
+
+.. image:: /docs/controls-overviews/images/control-system-hardware/openmesh-radio.png
+  :alt: OpenMesh OM5P-AN or OM5P-AC Radio
+  :width: 500
+
+Either the OpenMesh OM5P-AN or `OpenMesh OM5P-AC <https://www.andymark.com/products/open-mesh-om5p-ac-dual-band-1-17-gbps-access-point-radio>`__ wireless radio was used as the robot radio to provide wireless communication functionality to the robot. The device can still be configured as an Access Point for direct connection of a laptop for use at home. It can also be configured as a bridge for use on the field. The robot radio should be powered by one of the 12V/2A outputs on the VRM and connected to the roboRIO controller over Ethernet.
+
+
+
+##Programming your Radio
+
+This guide will show you how to use the FRC\ |reg|  Radio Configuration Utility software to configure your robot's wireless bridge for use outside of FRC events.
+
+##Prerequisites
+
+The FRC Radio Configuration Utility requires administrator privileges to configure the network settings on your machine. The program should request the necessary privileges automatically (may require a password if run from a non-administrator account), but if you are having trouble, try running it from an administrator account.
+
+Download the latest FRC Radio Configuration Utility Installer from the following links:
+
+`FRC Radio Configuration 24.0.1 <https://firstfrc.blob.core.windows.net/frc2024/Radio/FRC_Radio_Configuration_24_0_1.zip>`_
+
+`FRC Radio Configuration 24.0.1 Israel Version <https://firstfrc.blob.core.windows.net/frc2024/Radio/FRC_Radio_Configuration_24_0_1_IL.zip>`_
+
+.. note:: The _IL version is for Israel teams and contains a version of the OM5PAC firmware with restricted channels for use in Israel.
+
+.. note:: Teams planning on using the radio tool on a machine without the **2024** WPILib suite installed, will need a copy of Java installed. This can be downloaded `here <https://adoptium.net/temurin/releases/?version=17>`__.
```

Comment:
I think the link can be updated to https://adoptium.net/temurin/releases/?version=17&os=windows to filter on windows since the tool only runs on Windows